bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -671,7 +671,7 @@ case class TimeProblems(p: Proposal, s: ShellAdvisor) {
   lazy val requested = p.proposalClass.requestedTime
   def obsTimeSum(b: Band) = TimeAmount.sum(for {
     o <- p.observations if o.band == b
-    t <- o.time
+    t <- o.progTime
```

Comment:
This, admittedly, I was not entirely sure about, and I have it listed as a question in a list of several questions to be asked. I really more than anything here wanted to get a draft of this out as a "request for comments" and to get the discussion going as to what needs to be done moving forward, so for now I assumed the mapping from the old `time` to the new `progTime` when I wasn't certain and made a note in a text file to revisit this where it wasn't clear.